airflow/models/dag.py

```diff
@@ -388,6 +390,8 @@ def __exit__(self, _type, _value, _tb):
     @staticmethod
     def _upgrade_outdated_dag_access_control(access_control=None):
         """
+        _upgrade_outdated_dag_access_control
```

Comment:
It looks unreleated

Reply:
We should not be adding a dummy Summary heading for docstring @ktrueda